openfisca_us/variables/gov/states/md/tax/income/md_deductions.py

```diff
@@ -7,3 +7,18 @@ class md_deductions(Variable):
     label = "MD deductions"
     unit = USD
     definition_period = YEAR
```

Comment:
add reference - is this actually how salt is removed? we'll need to follow the law here